src/themes/greyVest/ErrorList.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
-import _ from 'lodash/fp'
 import ErrorBlock from './ErrorBlock'
 import ErrorText from './ErrorText'
 
-let ErrorList = ({ block = false, children }) =>
-  _.map(
-    e =>
-      block ? (
-        <ErrorBlock key={e}>{e}</ErrorBlock>
-      ) : (
-        <ErrorText key={e}>{e}</ErrorText>
-      ),
-    _.castArray(children)
-  )
+let ErrorList = ({ block = false, children, ...props }) =>
```

Comment:
Okay, but! To use that pattern outside `contexture-react`, we'd have to add `ErrorText` and `ErrorBlock` to our list of exports. I guess that's not really a big deal, but in this particular case reducing the API surface might be worth keeping the flag in there.

On the other hand, no reason not to add an ErrorComponent prop anyway, to allow for custom ones. I just don't want to give up `block` 😣

Reply:
If you're concerned about exporting `Error{Block,List}` you could use `ErrorList` by default and move `ErrorBlock` to bid-search or do the same with `ErrorList` and export only `ErrorBlock`. This is actually a wider discussion about composition vs configuration that we've had in the past btw. Also don't worry about exporting for now, which is actually another wider discussion around having a solid set of UI primitives that we can nicely compose (because we're not using a component library)